requirements/server-load-permission-requirements.md

```diff
@@ -15,10 +13,10 @@ While trying to load SharePoint farm settings with SPDocKit I received:
 
 #### Why is this warning occurring?
 
-During the Farm Load process SPDocKit will try to retrieve information about your servers. Please note that the following rights on your servers are optional, but depending on them some or all information about your server configuration and the corresponding best practice reports will not be available in SPDocKit. In order to allow SPDocKit to successfully gather all the information about your servers the following rights are necessary:
+During the Take Snapshot process SPDocKit will try to retrieve information about your servers. Please note that the following rights on your servers are optional, but depending on them some or all information about your server configuration and the corresponding best practice reports will not be available in SPDocKit. In order to allow SPDocKit to successfully gather all the information about your servers the following rights are necessary:
 
 * __Local Administrator__ on the server
-* Windows Update service up and running
+* __Windows Update service__ up and running
 
 #### What will these granted rights be used for?
 
```

Comment:
the __Local Administrator__ group
for WMI remote access (bez the)
see lower for what...